demo/src/main/java/com/orange/ods/demo/ui/about/AboutHtmlFileScreen.kt

```diff
@@ -34,17 +37,32 @@ fun AboutHtmlFileScreen(
     aboutItem?.let { item ->
         updateTopBarTitle(item.titleRes)
         val context = LocalContext.current
+        val configuration = LocalConfiguration.current
+        val horizontalPadding = dimensionResource(id = R.dimen.ods_screen_horizontal_margin).value
+        val verticalPadding = dimensionResource(id = R.dimen.ods_screen_vertical_margin).value
         AndroidView(
-            modifier = Modifier.padding(
-                horizontal = dimensionResource(id = R.dimen.ods_screen_horizontal_margin),
-                vertical = dimensionResource(id = R.dimen.ods_screen_vertical_margin)
-            ),
             factory = {
                 WebView(context).apply {
-                    webViewClient = WebViewClient()
+                    @SuppressLint("SetJavaScriptEnabled")
+                    settings.javaScriptEnabled = true
+                    webViewClient = object : WebViewClient() {
+                        override fun onPageFinished(view: WebView?, url: String?) {
+                            super.onPageFinished(view, url)
+                            view?.loadUrl("javascript:(function(){ document.body.style.padding = '${verticalPadding}px ${horizontalPadding}px' })();");
+                        }
+                    }
                     loadUrl("${FILE_PATH}${item.fileName}")
                     setBackgroundColor(Color.TRANSPARENT)
+                    if (WebViewFeature.isFeatureSupported(WebViewFeature.FORCE_DARK_STRATEGY)) {
```

Comment:
I tested on Galaxy S3 with Android 8.0. This device doesn't support `FORCE_DARK_STRATEGY` but supports `FORCE_DARK`. The webview content switches correctly from light to dark. The only difference is that the web view is automatically darkened (same as using `DARK_STRATEGY_USER_AGENT_DARKENING_ONLY`) instead of using our custom dark theme from the CSS. This is almost identical to what we have on Android 9.0 and later, except for the hyperlinks color. We won't be able to do anything better, but it's not a problem for me because this still fix the issue and the result is still satisfying.